public/js/test/mochitest/browser_dbg-editor-select.js

```diff
@@ -43,12 +30,12 @@ add_task(function* () {
   // Step through to another file and make sure it's paused in the
   // right place.
   yield stepIn(dbg);
-  assertPausedLocation(dbg, simple2, 2);
+  assertPausedLocation(dbg, "simple2.js", 2);
```

Comment:
I'd prefer to make anything that takes source support a string like this and passing the source object. I want to write some tests with eval sources soon and can't use urls like this. And here, since I already have the source objects, we could just use them.